Dockerfile

```
@@ -17,7 +17,7 @@ ENV ADDON_CATEGORIES="--private" \
     REPOS_FILE="odoo/custom/src/repos.yaml" \
     VERBOSE=0
 RUN apt-get update \
-    && apt-get install -yqq curl docker.io git jq \
+    && apt-get install -yqq curl docker.io git jq build-essential python3-dev \
```

```suggestion
    && apt-get install -yqq curl docker.io git jq build-essential \
```

These should be available in the image already, and installing the apt version would conflict.